examples/hackweek_demo/plot_pnl.py

```diff
@@ -52,20 +72,45 @@ def calculate_spot_price_for_position(
 
 
 def calculate_current_pnl(pool_config: pd.Series, pool_info: pd.DataFrame, current_wallet: pd.DataFrame) -> pd.Series:
-    """Calculates the most current pnl values."""
+    """Calculate the most current pnl values.
+
+    Calculate_spot_price_for_position calculates the spot price for a position that has matured by some amount.
+
+    Arguments
+    ---------
+    pool_config : pd.Series
+        Time-invariant pool configuration.
+    pool_info : pd.DataFrame
+        Pool information like reserves. This can contain multiple blocks, but only the most recent is used.
+    current_wallet : pd.DataFrame
+        Current agent holdings indexed by address and position (LP, LONG-YYYYMMDD, SHORT-YYYYMMDD, etc.)
+
+    Returns
+    -------
+    pd.Series
+        Calculated pnl for each row in current_wallet.
+    """
     # pylint: disable=too-many-locals
     # Most current block timestamp
     latest_pool_info = pool_info.loc[pool_info.index.max()]
     block_timestamp = latest_pool_info["timestamp"].timestamp()
 
-    # Calculate for base
+    # Base is valued at 1:1, since that's our numéraire (https://en.wikipedia.org/wiki/Num%C3%A9raire)
     base_pnl = current_wallet[current_wallet["baseTokenType"] == "BASE"]["tokenValue"]
 
-    # Calculate for lp
+    # LP value = users_LP_tokens * sharePrice
```

Comment:
This is really nice, thanks.  Can you also throw a quick summary in the sphinx docs too?  Something like:
```
pnl = user_long_value + user_short_value + user_lp_value + user_withdraw_share_value
```
if we aren't doing that let's add a ::todo to fix it.

Reply:
looks like our docs only cover things in `/elfpy` not in `/examples` or did you mean something else? https://elf-simulations-git-explaincalculatecurrentpnl-delvtech.vercel.app/